server/trucking/src/main/java/by/bsu/famcs/trucking/front/controller/CargoController.java

```diff
@@ -0,0 +1,33 @@
+package by.bsu.famcs.trucking.front.controller;
+
+import by.bsu.famcs.trucking.back.entity.CargoBack;
+import by.bsu.famcs.trucking.service.CargoService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin(origins = {"https://trucking-famcs.herokuapp.com"})
+@RestController
+public class CargoController {
+
+    @Autowired
+    private CargoService cargoService;
+
+    @GetMapping(path = "/users/{id}/cargo",
```

Comment:
We figured out that if the user has authorised and know the id of another user he can use this query to get cargoes of another user. But it can be solved by removing {id} from URL and getting it from authorisation info.